app/models/user.rb

```diff
@@ -2,5 +2,5 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments
 
-  validates :username, presence: true, length: { minimum: 3 }
+  validates :username, presence: true, uniqueness: true, length: { minimum: 3 }
 end
```

Comment:
## 1. Behaviour
- [ ] Make sure the comment can be created properly. **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingComment.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails)). Currently the validations are there but there is a missing column in the comments table that should enforce this behaviour ***(Hint: We are lacking the `post_id` column in this table)***

## 2. General
- [ ] Let's unsure that our project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

- [ ] In our next iteration, we need to ensure that we commit as often as possible. It makes the workflow easier to track changes being added and track down bugs easily. We need to ensure also the pull request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [ ] While at it we also ensure that our commits are more descriptive ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [ ] Kindly work on the linter errors and ensure they all passing all the checks.  Thank you.